src/jit.rs

```diff
@@ -1619,73 +1623,32 @@ impl JitCompiler {
             (AccessType::Store, 8i32),
         ] {
             let target_offset = len.trailing_zeros() as usize + 4 * (*access_type as usize);
-            let stack_offset = if !self.config.dynamic_stack_frames && self.config.enable_stack_frame_gaps {
-                24
-            } else {
-                16
-            };
 
             self.set_anchor(ANCHOR_MEMORY_ACCESS_VIOLATION + target_offset);
```

Comment:
Do we still need a separate `ANCHOR_MEMORY_ACCESS_VIOLATION` anchor for all 8 generic specializations? Why not just keep the error returned by

Reply:
nope, removed